test/e2e/install/install_test.go

```diff
@@ -38,12 +38,14 @@ import (
 	"github.com/openshift-kni/numaresources-operator/pkg/status"
 	e2eclient "github.com/openshift-kni/numaresources-operator/test/utils/clients"
 	"github.com/openshift-kni/numaresources-operator/test/utils/configuration"
+	"github.com/openshift-kni/numaresources-operator/test/utils/crds"
 	"github.com/openshift-kni/numaresources-operator/test/utils/machineconfigpools"
 	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
 )
 
 const (
-	crdName      = "noderesourcetopologies.topology.node.k8s.io"
+	crdNRTName   = "noderesourcetopologies.topology.node.k8s.io"
+	crdNROName   = "numaresourcesoperators.nodetopology.openshift.io"
```

Comment:
not sure these belongs here or in the new `crds` package. But it's minor.

Reply:
yeah you are totally right ... will change it